### PR TITLE
chore: hide plugins for now

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright-test/src/loader.ts [hunk 1]
```diff
@@ -142,8 +124,6 @@ export class Loader {
       (config as any).screenshotsDir = path.resolve(configDir, (config as any).screenshotsDir);
     if (config.snapshotDir !== undefined)
       config.snapshotDir = path.resolve(configDir, config.snapshotDir);
-    if (config.webServer)
```
Comment (dgozman) [inline packages/playwright-test/src/loader.ts:145]: Why removing this?

File: packages/playwright-test/src/plugins/index.ts [hunk 2]
```diff
@@ -13,5 +13,28 @@
...
+export interface TestRunnerPlugin {
+  name: string;
+  setup?(config: FullConfig, configDir: string, rootSuite: Suite): Promise<void>;
+  teardown?(): Promise<void>;
+}
+
 export { webServer } from './webServerPlugin';
 export { gitCommitInfo } from './gitCommitInfoPlugin';
+
+let runnerInstanceToAddPluginsTo: Runner;
```
Comment (dgozman) [inline packages/playwright-test/src/plugins/index.ts:30]:
```suggestion
let runnerInstanceToAddPluginsTo: Runner | undefined;
```

**Review by dgozman — APPROVED** [review]

File: packages/playwright-test/src/loader.ts (hunk 1 above)
Comment (pavelfeldman) [reply packages/playwright-test/src/loader.ts:145]: `this._options.cwd = this._options.cwd ? path.resolve(configDir, this._options.cwd) : configDir;` is also in the plugin setup. Looks redundant.

File: packages/playwright-test/src/DEPS.list [hunk 3]
```diff
@@ -3,4 +3,5 @@
 matchers/
 reporters/
 third_party/
+plugins/
 plugins/webServerPlugin.ts
```
Comment (rwoll) [inline packages/playwright-test/src/DEPS.list:7]:
```suggestion
plugins/
```

**Review by rwoll — APPROVED** [review]
LGTM. Wait for the bots as reporter-html.spec.ts might complain.

File: examples/components-vue/playwright.config.ts [hunk 4]
```diff
@@ -1,18 +1,10 @@
...
   retries: process.env.CI ? 2 : 0,
-  reporter: process.env.CI ? [
-    ['html', { open: 'never' }],
-  ] : [
-    ['html', { open: 'on-failure' }]
-  ],
-  plugins: [
-    viteCT()
-  ],
+  reporter: 'html',
```
[the remark this answers is not in the document]
Comment (pavelfeldman) [reply examples/components-vue/playwright.config.ts:7]: I'm just making it in-sync with our default create-playwright.

**Pull request merged**